### ISSUE TITLE
Possible error in documentation fiber code examples

### ISSUE THREAD

**Issue by Goju-Ryu** [fubark/cyber#85]
When reading the concurrency section on fibers I had some trouble understanding why `fiber.reset()` and `fiber.status()` was called in example only defining a fiber called `task`. After reading through some documentation and testing in the playground I have come to the conclusion that it is probably an error of the documentation. 

The examples in question are the following:
https://github.com/fubark/cyber/blob/f3bda254d2f84ba0b7bc7667920c75c57839ded4/docs/docs.md?plain=1#L2364-L2402

**Comment by fubark** [comment]
Yep, that's a typo. Also, `Fiber.reset` hasn't been implemented.
